app/assets/javascripts/lajoya.js

```diff
@@ -139,7 +132,8 @@
     }
 
     navigator.getUserMedia({audio: true}, startUserMedia, function(e) {
-      alert('Favor de usar Chrome, actualizar la página, y permitir acceso al micrófono');
+      alert('Favor de usar Chrome, actualizar la página, y permitir acceso al'\
+        ' micrófono');
```

Comment:
Expected ')' and instead saw ' micrófono'.<br>Missing semicolon.<br>Expected an identifier and instead saw ')'.<br>Expected an assignment or function call and instead saw an expression.